Formula/w/webkitgtk.rb

```diff
@@ -1,8 +1,8 @@
 class Webkitgtk < Formula
   desc "GTK interface to WebKit"
   homepage "https://webkitgtk.org"
-  url "https://webkitgtk.org/releases/webkitgtk-2.46.2.tar.xz"
-  sha256 "5aae1c5c0a30d5e9c5831652b6f1bdefc31d75e0ad81cb40185b0aed92ce79b6"
+  url "https://webkitgtk.org/releases/webkitgtk-2.46.3.tar.xz"
```

Comment:
Skipping for now as conflict with https://github.com/Homebrew/homebrew-core/pull/193114